modules/cudev/test/test_main.cpp

```diff
@@ -43,4 +43,4 @@
 
 #include "opencv2/ts.hpp"
 
-CV_TEST_MAIN("cv")
```

Comment:
This line should not be changed. The cudev tests are slightly different from the other cuda* tests.